### DISCUSSION TITLE
Next JS 13.0.3 a client-side exception has occurred

### DISCUSSION THREAD

**Discussion by rohanbhangui** [vercel/next.js#42956] [Help]
Hi,

I upgraded from nextjs 13.0.1 to 13.0.3 recently and I started to get an error when deploying to netlify.

Next JS 13.0.3 a client-side exception has occurred. I get an internal 500 server error.

https://deploy-preview-9--boring-curran-8a0702.netlify.app/

I can compile, build and run product build locally but deploying is the issue. I check the build log and didnt see any errors.

I am trying to use an error boundary but havent gotten the error to popup. Any ideas?

**Comment by icyJoseph** [comment]
Looks like you've fixed it?
